sssom/writers.py

```diff
@@ -14,6 +13,7 @@
 from rdflib import Graph, URIRef
 from rdflib.namespace import OWL, RDF
 
+from . import SCHEMA_YAML
```

Comment:
this might cause circular imports. better to make a constants file.

Reply:
there is a constants.py.. you can use that

Reply:
That is in `tests`

Reply:
I created a new one in `sssom`